webrtc.html

```diff
@@ -8942,6 +8956,54 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+        The <dfn>stats selection algorithm</dfn> is as follows:
+        <ol>
+          <li>
+            Let <var>result</var> be an empty RTCStatsReport.
+          </li>
+          <li>If <var>selector</var> is <code>null</code>, gather stats for the
+            whole <var>connection</var>, add them to <var>result</var>, return
+            <var>result</var>, and abort these steps.
+          </li>
+          <li>If <var>selector</var> is an <a>RTCRtpSender</a>, gather stats for
+            and add the following objects to <var>result</var>:
+            <ul>
+              <li>
+              All <code>RTCInboundRTPStreamStats</code> objects corresponding to
+              <var>selector</var>.
+              </li>
+              <li>
+              All stats objects referenced directly or indirectly by the
+              <code>RTCInboundRTPStreamStats</code> objects added.
+              </li>
+            </ul>
+          </li>
+          <li>If <var>selector</var> is an <a>RTCRtpReceiver</a>, gather stats
+            for and add the following objects to <var>result</var>:
+            <ul>
+              <li>
+              All <code>RTCOutboundRTPStreamStats</code> objects corresponding
+              to <var>selector</var>.
+              </li>
+              <li>
+              All stats objects referenced directly or indirectly by the
+              <code>RTCOutboundRTPStreamStats</code> added.
+              </li>
+            </ul>
+          </li>
```

Comment:
Great! Compare to [mine](https://docs.google.com/presentation/d/1sfVD5VAHg1K7Hee37kooAbGEajfNIhaKUWmbIRZoioI/). The list is [here](https://github.com/w3c/webrtc-stats/issues/116#issuecomment-279084420) and is deterministic, so all the info should be known upfront about which stats to gather.

I count `RTCPeerConnectionStats`, `RTCDataChannelStats` and `RTCMediaStreamStats` as types not included. But more importantly is the multiple instances related to other senders/receivers a selector filters out.